custom-namespaces.yaml

```diff
@@ -435,7 +435,7 @@ mozilla_vpn:
       type: table_view
       tables:
         - channel: release
-          table: mozdata.mozilla_vpn.site_metrics_summary
+          table: mozdata.mozilla_vpn.site_metrics_summary_v2
```

Comment:
yes that is correct normally, but this one is a bit of a unique case.  Because we are trying to switch over from GA3 to GA4 but also leave the ability to quickly switch back if needed, we didn't update the view itself to point to the new table, but rather are just switching out the looker view reference.

Reply:
Wouldn't it be easier to make this change inside the view then?